diff --git a/node_modules/@jamsch/expo-speech-recognition/ios/ExpoSpeechRecognitionModule.swift b/node_modules/@jamsch/expo-speech-recognition/ios/ExpoSpeechRecognitionModule.swift
index cb4b046..cf52b42 100644
--- a/node_modules/@jamsch/expo-speech-recognition/ios/ExpoSpeechRecognitionModule.swift
+++ b/node_modules/@jamsch/expo-speech-recognition/ios/ExpoSpeechRecognitionModule.swift
@@ -96,7 +96,7 @@ public class ExpoSpeechRecognitionModule: Module {
       }
       permissions.askForPermission(
         usingRequesterClass: EXSpeechRecognitionPermissionRequester.self,
-        resolve: promise.resolver,
+        resolve: promise.legacyResolver,
         reject: promise.legacyRejecter
       )
     }
@@ -107,7 +107,7 @@ public class ExpoSpeechRecognitionModule: Module {
       }
       permissions.getPermissionUsingRequesterClass(
         EXSpeechRecognitionPermissionRequester.self,
-        resolve: promise.resolver,
+        resolve: promise.legacyResolver,
         reject: promise.legacyRejecter
       )
     }
